assets/css/less/wpbdp/abstracts/variables/typography.less

```diff
@@ -0,0 +1,7 @@
+// Font Size
+@font-size-body:  16px;
```

Comment:
Are these variables still needed? Or should this file be removed for now?

Reply:
I will apply them to particular styles.

Reply:
I've removed this file.